packages/core/src/util/url.js

```diff
@@ -15,6 +15,9 @@ const secrets = require('../secrets');
 exports.sanitizeUrl = function sanitizeUrl(urlString) {
   let normalizedUrl;
   try {
+    if (!URL.canParse(urlString)) {
```

Comment:
If an invalid URL is encountered, for example, when only a path is provided without a host portion, this scenario is accounted for.